cmd/server/assets/header.html

```diff
@@ -187,23 +191,23 @@
 {{define "navbar"}}
 <header class="mb-3">
   {{if .currentRealm}}
-  <div href="/" class="d-block px-3 py-2 text-center text-bold text-white realm-header">
+  <div href="/" class="d-block px-3 py-2 text-center text-bold text-white bg-primary">
     {{.currentRealm.Name}}{{if .currentRealm.RegionCode}} - {{.currentRealm.RegionCode}}{{end}}
   </div>
   {{end}}
 
-  <nav class="navbar navbar-expand-md navbar-dark bg-primary">
+  <nav class="nav nav-tabs navbar-expand-md navbar-light bg-light">
     <div class="container">
       {{template "navtoggle" .}}
 
-      <div class="collapse navbar-collapse" id="navbar">
+      <div class="collapse navbar-collapse mt-2" id="navbar">
         {{if .currentRealm}}
-        <ul class="navbar-nav mr-auto">
+        <ul class="nav mr-auto">
           <li class="nav-item">
-            <a class="nav-link" href="/home">Issue code</a>
+            <a class="nav-link {{if .homeTab}}active{{end}}" href="/home">Issue code</a>
```

Comment:
This feels subtle and could easily break if someone doesn't set the right template vars. I'd rather do this in js or using the `currentpage` thing I created earlier.

Reply:
I made _a_ generic currentpage thing... but where can I find the one created earlier - I don't want to duplicate work.

I don't like using javascript because it tends to load after the page has been rendered. Plus it seems unnecessary